test/helpers/story/drivers/dapp-story-test-driver.ts

```diff
@@ -132,40 +149,45 @@ export default class DappStoryTestDriver extends StoryTestDriver {
     const dapp = action.actionType
     switch (dapp) {
       case 'UNISWAP': {
-        const newTest = new Test('UNISWAP Swap DAPP', (async () => {
+        const newTest = new Test('UNISWAP Swap DAPP', async () => {
           const borrower = await getNamedSigner('borrower')
           const loan = await LoanStoryTestDriver.getLoan(hre, borrower)
           let shouldPass = true
           //read the state and determine if this should pass
-          if(!loan) shouldPass = false
+          if (!loan) shouldPass = false
           if (shouldPass) {
             await DappStoryTestDriver.swapUniSwap(hre, loan)
           } else {
-            await expect(await DappStoryTestDriver.swapUniSwap(hre, loan)).to.be.reverted
+            await expect(await DappStoryTestDriver.swapUniSwap(hre, loan)).to.be
+              .reverted
           }
-        }))
+        })
         tests.push(newTest)
         break
       }
       case 'SUSHISWAP': {
-        const newTest = new Test('SUSHISWAP Swap DAPP', (async () => {
+        const newTest = new Test('SUSHISWAP Swap DAPP', async () => {
           const borrower = await getNamedSigner('borrower')
           const loan = await LoanStoryTestDriver.getLoan(hre, borrower)
-          let shouldPass = true
+          let shouldPass = false
           //read the state and determine if this should pass
           if (!loan) shouldPass = false
           const { details, diamond } = loan
           const escrowAddress = await diamond.getLoanEscrow(details.loan.id)
-          const lendingBalBefore = await details.lendingToken.balanceOf(escrowAddress)
-          if(lendingBalBefore.lte(0)) shouldPass = false
+          const lendingBalBefore = await details.lendingToken.balanceOf(
+            escrowAddress
+          )
+          if (lendingBalBefore.lte(0)) shouldPass = false
           if (shouldPass) {
             await DappStoryTestDriver.swapSushiSwap(hre, loan)
           } else {
-            await DappStoryTestDriver.swapSushiSwap(hre, loan).catch((error) => {
-              expect(error).to.exist
-            })
+            await DappStoryTestDriver.swapSushiSwap(hre, loan).catch(
+              (error) => {
+                expect(error).to.exist
+              }
+            )
           }
```

Comment:
when are we testing `shouldPass === true` ?

Reply:
i want to merge this into currently existing sushiswap  branch